index.html

```diff
@@ -4,19 +4,66 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-  <link rel="stylesheet" href="css/style.css">
+  <title>Peronal Portfolio</title>
+
+  <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-    <p>Wello World!</p>
-    <script  src="js/index.js"></script>
-</body>
-</html>
-
-
+  <main id="main">
+    <header>
+      <nav id="navbar">
+        <div class="navbar-buttons">
+         <div class="navbar-button Home">
+            Home
+         </div>
+         <div class="navbar-button About">
+            About
+         </div>
+         <div class="navbar-button Blog">
+            Blog
+         </div>
+        </div>
+      </nav>
+    </header>
+    <!-- section welcome -->
+    <div id="welcome-section">
+      <h1 class="welcome-h1" >Hey I am Jose Delphin</h1>
+      <h2 class="welcome-h2">a web developer</h2>
+    </div>
+    <!-- section project -->
+      <div id="section-projects">
+        <h1 class="projects-h1">These are some of my projects</h1>
+        <div class="projects">
 
 
 
 
 
+        </div>
+      </div>
+      <!-- section contact -->
+      <div class="section-contact">
+        <h1 class="contact-h1">Lets's work together...</h1>
+        <h2 class="contact-h2">How do you take your coffee?</h2>
+        <div class="socailMedia-Div">
+          <div class="facebook a">
+            <div class="hvrFacebook">Facebook</div>
+          </div>
+          <div class="github a">
+            <div class="hvrGithub">GitHub</div>
+          </div>
+          <div class="twitter a">
+            <div class="hvrTwitter">Twitter</div>
+          </div>
+          <div class="mail a">
+            <div class="hvrMail">Send a mail</div>
+          </div>
+          <div class="callMe a">
+            <div class="hvrCallme">Call me</div>
+          </div>
+        </div>
+        <div class="red-line"></div>
+      </div>
```

Comment:
Please wrap this into a section tag.  Since it is a semantic element for creating standalone sections in a web page

Reply:
- [OPTIONAL] I think , as suggested by the previous reviewer, It would be better to use some emojies to make it human-friendly and cool :sunglasses: